web/src/WifiNetworkListItem.test.jsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+import { screen } from "@testing-library/react";
+import { installerRender } from "./test-utils";
+
+import WifiNetworkListItem from "./WifiNetworkListItem";
+
+jest.mock("./WifiConnectionForm", () => () => "WifiConnectionForm mock");
+jest.mock("./WifiNetworkMenu", () => () => "WifiNetworkMenu mock");
+
+const onSelectCallback = jest.fn();
+const fakeNetwork = {
+  ssid: "Fake Wi-Fi AP",
+  security: ["WPA2"],
+  strenght: 86
+};
+
+const fakeSettings = {
+  wireless:  {
+    password: "notSecret"
+  }
+};
+
+describe("NetworkListItem", () => {
+  it("renders an input radio for selecting the network", async () => {
```

Comment:
In general, there is no need for async (except where user interaction is tested).